packages/components/psammead-amp-geo/src/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eea: [
```

Comment:
Could this list of countries also be accesible via an export? Could be useful if people want a list of EEA countries?

Reply:
Once we have a second use-case for a list of EEA countries, sure. I don't believe there is one at the moment.